tests/raw_query/models.py

```diff
@@ -1,8 +1,10 @@
 from django.db import models
 from django_mongodb.manager import MongoManager
+from django_mongodb.fields import ObjectIdAutoField
 
 
 class Author(models.Model):
+    id = ObjectIdAutoField(primary_key=True)
```

Comment:
You don't need to add an explicit primary key to all these methods. Put `DEFAULT_AUTO_FIELD = "django_mongodb.fields.ObjectIdAutoField"` in your test settings file.